app/controllers/hackathons/submissions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Hackathons::SubmissionsController < ApplicationController
+  def index
+    @hackathons = Hackathon.not_approved.where applicant: Current.user
+
+    redirect_to new_hackathons_request_path if @hackathons.none?
+  end
+
+  def new
+    @hackathon = Hackathon.new
+  end
+
+  def create
```

Comment:
To submit a form, the user should not need to login.

We'll take in an email address field, which we'll use to `find_or_create` a User. (It would be nice to have the user's name as well; not required)